dashboard/app/models/script_level.rb

```diff
@@ -37,6 +36,22 @@ def script
     Script.get_from_cache(script_id)
   end
 
+  def level
+    levels[0]
+  end
+
+  def level=(l)
+    levels[0] = l
+  end
+
+  def level_id
+    levels[0].id
+  end
+
+  def level_id=(new_level_id)
+    levels[0] = Level.find(new_level_id)
+  end
```

Comment:
Ah, then this seems perfectly reasonable as a temporary measure. Could you add a comment with a `TODO`?